powerline.el

```diff
@@ -23,6 +23,10 @@
 
 (require 'cl-lib)
 
+(defcustom powerline nil
```

Comment:
does this do anything?

Reply:
That was supposed to be `defgroup`. I have amended a fix.